*** tests/A-16-02-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-16-02-M-ok.outhyp	2021-11-12 18:07:47.985639000 +0000
***************
*** 94,100 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|250|0|25|25
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 94,100 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|25|25
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
