*** tests/A-17-01-M-ok.out	2021-11-10 23:23:36.000000000 +0000
--- tests/A-17-01-M-ok.outhyp	2021-11-12 00:44:41.545950000 +0000
***************
*** 17,24 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Mostrar compras com parceiro: Operação inválida: O parceiro 'M4' não existe.
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 17,23 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
