*** tests/A-01-03-M-ok.out	Tue Oct 26 14:38:06 2021
--- tests/A-01-03-M-ok.outhyp	Wed Oct 27 03:25:36 2021
***************
*** 28,37 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: E4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
! P1|Lages|Lisboa, Portugal|NORMAL|0|0|0|0
! R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 28,34 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
