*** tests/A-01-04-M-ok.out	Wed Oct 27 16:27:27 2021
--- tests/A-01-04-M-ok.outhyp	Wed Oct 27 18:23:39 2021
***************
*** 15,31 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|200|5000
! OXIGENIO|1200|2500
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|S1|200|5000
! OXIGENIO|P1|1200|2500
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
--- 15,27 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
***************
*** 72,78 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 68,74 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
